### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (2 - Model based guidance)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#481]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

1)  Model-based guidance (top down  without simultaneous bottoms up)
I was going to put the Model Based approach in the Laudits category above.  It's good to have a theory or model behind what you do.  
However, all models need to be also data-based rather than just theory-based to be useful.  That is, they need to model “what has to happen” and “what can happen” not “what we wish was possible”.  Thus, good model building involves both a top-down and a simultaneous bottom up look at the topic.  A bottom-up approach requires that you first gather all of the things that you are trying to model or organize and study them before you start trying to draw top-down.  
You can't design an organization, or a structure, or a model for something until you actually first look at all of the parts of what you trying to model.  In this case, the group should have a full set of the provisions for the new standard – or as many as possible.  Not only the old provisions but also all of the new provisions on the table that they are trying to create a model for.  
If this had been done, (and the rules for qualifying a provision were applied) some of the problems cited below may have been avoided.  As it is, the document spends all of its time laying out it's elaborated model for provisions and testing etc. without having the provisions and tests in hand.  
As a result, everything may seem to fit a model when in fact it does not when you actually put things in it.  In some ways it reminds me of Marie Kondo who says don't organize your closet until you first empty it all out, sort it and see what you have.  Otherwise, you end up with an organization that doesn't actually fit the clothes that you have in number quantity and type.  In this case we are not talking about sorting, (it would be nice if it was such a simple task) but something much more complicated.  The underlying principle however is the same in all of science.  Models that are created without data are of limited value in practice.  Models that are not tested by applying them against data are speculative.
It is also important to remember the quote by George Box. 
“All models are wrong, but some are useful”.
That is, a model can be useful in guiding your work, but adherence to the model often causes you to force things to fit that don't really fit.  In WCAG 2.0 we also created models for provisions and, for example, what would be in level A, AA, and AAA.   However, when we actually came to assigning provisions to the different levels using our model/rules- we found that the model/rules didn't work.  We kept adding more and more factors, but it still never worked.  In the end we found that the data (the provisions) and the reality of the needs of different groups, as well as the interaction between different provisions informed the decisions.  As a result, the answer to the question "what were the factors determining which level something when into" ended up being a list of factors that were used, but no strict formula -- because no strict formula could account for the complexity.
Similarly, the models in WCAG 3.0 may be good for thinking – but are unlikely to work as planned when you actually try to apply them to all of the provision – and the constraints in creating provisions.   Suggest that, with this modeling phase being done – you focus next on collecting all of the provisions and seeing if they fit your model – or you need to rethink, revise. 
Top-down and Bottom-up in combination. 
Actually the best approach usually is to do both top-down and bottom-up together.  Since you already have WCAG and the work of the COGA group – you have a first pass at bottom up. You now also have a first pass at top-down.   The next step is probably to gather all of your potential provisions (from WCAG and COGA and other disability efforts, then look at them and come up with your first-final set of guidelines.   (if you want a document that can be used in regulatory work – you would have to run them all through a filter to get a set of objective, testable, items.  If going the guidance route – you can include them all.)     You can then take these and fit them into your model to see if it works.  If not – you can modify the model as needed.  And the iterate on provisions, model, and the language / wording for both.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
